src/button.js

```diff
@@ -3,20 +3,31 @@ import PropTypes from 'prop-types'
 import { makeStyles } from '@material-ui/core/styles'
 import MUIButton from '@material-ui/core/Button'
 import clsx from 'clsx'
+import Test from './theme/overrides/Test'
 
-const useStyles = makeStyles((theme) => {
-  return {
-    label: {
-      color: 'rgba(0, 0, 0, 0.0) !important',
-    },
-    noSpacing: {
-      padding: theme.spacing(0),
-    },
-  }
-})
+const useStyles = color => {
+  const selectStyle = makeStyles((theme) => {
+    const customColor = Test(color)
+
+    return {
+      contained: customColor.contained,
+      containedPrimary: customColor.containedPrimary,
+      outlinedPrimary: customColor.outlinedPrimary,
+      textPrimary: customColor.textPrimary,
+      label: {
+        color: 'rgba(0, 0, 0, 0.0) !important',
+      },
+      noSpacing: {
+        padding: theme.spacing(0),
+      },
+    }
+  })
+
+  return selectStyle()
```

Comment:
with `makeStyles()`, MUI creates a new class instead of overriding its default class. 
![Screen Shot 2021-03-25 at 5 13 34 PM](https://user-images.githubusercontent.com/59836949/112544516-78b58800-8d8d-11eb-9df8-e04ecbf27eb0.png)

One way to work around this issue would be appending a class inside the button component which points the actual class with `&` selector.

```jsx
const useStyles = makeStyles(()=>{
  button: {
    fontSize: 12,
    '&.containedPrimary': {
      background: '#0075FF',
    }
  }
}
(...)
return <Button className={classes.button}/>